content/en/continuous_delivery/deployments/argocd.md

```diff
@@ -169,10 +169,46 @@ metadata:
     dd_k8s_cluster: example-cluster
 ```
 
+
 ## Visualize deployments in Datadog
 
 The [**Deployments**][6] and [**Executions**][7] pages populate with data after a deployment is executed. For more information, see [Search and Manage][9] and [CD Visibility Explorer][10].
 
+
+## Correlate deployments with CI pipelines
+
+If Argo CD deployments are triggered from a CI pipeline, you may correlate the deployment execution and the pipeline,
+allowing to check the associated deployments to a pipeline and the pipeline that triggered a deployment from Datadog. See the [section below](#visualize-correlated-pipelines-and-deployments) to see how the visualization looks in Datadog.
+
+In this moment, not all the setups are supported. To be able to make the correlation your setup needs to make changes to the repository Argo CD is monitoring using `git commit` in your CI. The following diagram represents an example of this kind of setup:
```

Comment:
Based on my comments above, I'd mention the use case we support at the beginning (app repo, config repo, automated commit) and remove this sentence